src/tests/ftest/control/ms_resilience.py

```diff
@@ -189,23 +190,23 @@ def launch_servers(self, resilience_num):
 
         return replicas
 
-    def kill_servers(self, leader, replicas, N):
+    def kill_servers(self, leader, replicas, num_hosts):
         """Kill a subset of servers in order to simulate failures.
 
         Args:
             leader (NodeSet): hostname of current leader.
             replicas (NodeSet): replica hostnames.
-            N (int): Number of hosts (including leader) to stop.
+            num_hosts (int): Number of hosts (including leader) to stop.
 
         Returns:
             NodeSet: hosts that were stopped.
 
         """
-        kill_list = NodeSet.fromlist(random.sample(list(replicas), N))
+        kill_list = NodeSet.fromlist(random.sample(list(replicas), num_hosts))
```

Comment:
Any idea why we don't need a "nosec" for this random call?

Reply:
Apparently, `random.sample` isn't on the blacklist:
# pylint: disable-next=relative-beyond-top-level